konrad/radiation/arts.py

```diff
@@ -312,7 +274,7 @@ def calc_optical_thickness(self, atmosphere, t_surface):
             axis=-1,
         )
 
-        return self.ws.f_grid.value.copy(), tau
+        return self.ws.f_grid.value.value.copy(), tau
```

Comment:
Same here.

Reply:
Are you referring to these three options? 
```py
ws.f_grid.value.value
ws.f_grid.value.value.copy()
ws.f_grid.value.value[:].copy()
```

I would argue that the first option differs from the others because it only keeps the reference to the variable in memory which, depending on the use case, can cause unwanted side effects (e.g. loops over scenarios).

The 2. and 3. options have the same behaviour (as it does not matter if we say `ndarray` or `ndarray[:]`. Do you prefer the latter because it clearly shows that we are dealing with the NumPy representation?

Reply:
I was referring to these three options:
ws.f_grid.value[:].copy()
np.copy(ws.f_grid.value)
ws.f_grid.value.value.copy()

The advantage of the first two is that you don't need the double `.value.value`.

Reply:
Oh I missed the first one! (Actually you doubled the `.value` in your example in the thread above 😉)

I agree that `ws.f_grid.value[:].copy` seems to be most readable of the options 👍

Reply:
> (Actually you doubled the `.value` in your example in the thread above 😉)

🤦‍♂️ Sorry. 😂